### DISCUSSION TITLE
Is dotnet/nightly/aspnet:6.0.19-jammy-chiseled-amd6 would be no longet updated?

### DISCUSSION THREAD

**Discussion by Bafff** [dotnet/dotnet-docker#4750] [Q&A]
Hi!

I read through the support policy but still struggling to understand it in my case.

Since .NET 6.0.20 was released there would be no base image (i.e. OS patches) updates to this image `dotnet/nightly/aspnet:6.0.19-jammy-chiseled-amd6`? Only newer, `dotnet/nightly/aspnet:6.0.20-jammy-chiseled-amd6` would be updated it's base image several time until 6.0.20 would be supported?

So probably that's why I already d not see this .NET version as supported here?
![image](https://github.com/dotnet/dotnet-docker/assets/26187677/cb4cd552-b2ab-4e38-81eb-4e6e8818e9d4)

If this is the case, how would you suggest, without going with the floating tag we should be notified/react on the new .NET images, so we would not end up being on some tag forever just because we missed that .NET team is working hard on new builds :) We are a small team so there are not a lot of resources designated for security.

**Comment by richlander** [comment]
I think you've mostly answered your own question. Why are you using `nightly`?

See:

<img width="763" alt="image" src="https://github.com/dotnet/dotnet-docker/assets/2608468/3de8ecb0-b9ef-44be-bfb7-1cb8d33abbb3">

**Reply by Bafff** [reply]
I do not see it here without using `nightly` for .NET 6.0 :)

https://mcr.microsoft.com/en-us/product/dotnet/aspnet/tags
![image](https://github.com/dotnet/dotnet-docker/assets/26187677/f462d02a-0198-4016-8940-f19aae6e212d)

And as I understand until chiseled images would not be GA'd they would not leave /nightly/. So I even not sure how you can made this screenshot :)

**Comment by richlander** [comment]
I see. The screenshot is from nightly. Anyway, my point was to show you the range of tags available.

**Comment by mthalman** [comment]
Image names in the `nightly` repo are for testing/evalution purposes. They are not officially supported. And yes, that means Chiseled images for .NET 6 are not yet officially supported because those tags are not available outside of the `nightly` repo.

As for the use of the patch version in the tag name, you do need to keep that updated each month to remain on a supported version. As [.NET's support policy states](https://dotnet.microsoft.com/platform/support/policy/dotnet-core), you must be using the latest patch version to have a supported version of the product. New patch versions of .NET are released on a predictable schedule on the 2nd Tuesday of each month. To avoid needing to update the tag you reference each month, you can use a "floating" tag (`6.0` instead of `6.0.20`) which will always reference the latest patch version.
